cpp/tensorrt_llm/batch_manager/kvCacheManager.cpp

```diff
@@ -2450,12 +2450,48 @@ BlocksPerWindow BaseKVCacheManager::calculateMaxNumBlocks(executor::KvCacheConfi
     };
 
     std::map<SizeType32, float> windowSizeToShare;
-    // NOTE: Righteously, blocks allocated should be proportional with
-    // regard to window size. Currently, we are first allocating identical
-    // number of blocks for all layers to achieve identical performance.
-    for (auto const& [windowSize, _] : windowSizeToLayers)
+    if (auto envStr = std::getenv("TRTLLM_WINDOW_SIZE_SHARES"))
```

Comment:
Could we add some comments about the expected format of the `TRTLLM_WINDOW_SIZE_SHARES` env var and what it does? It may not be that obvious from the code below. Thanks